input/pagecontent/l3_libraries.md

```diff
@@ -9,24 +9,100 @@ Libraries are FHIR Artifacts containing the definitions of data querying and pro
 
 * Library resources - populated with the necessary metadata and content
 
+#### **Metadata**
+
+##### **Shareable**
+
+* Library resources SHALL conform to CRMIShareableLibrary
+
+| Element | Guidance |
+| ---- | ---- |
+| id | SHALL be populated with {{Library Name}}
+| url | SHALL be populated with {{canonical base}}/Library/id |
+| version | SHALL be populated with {{ig version}} |
+| versionAlgorithm | SHOULD be populated with {{ig version algorithm}} |
+| versionPolicy | SHOULD be populated with {{strict | metadata}} |
+| name | SHALL be populated with {{Library Name }} |
+| title | SHALL be populated with {{Library Title }} |
+| status | SHALL be populated with {{ig status}} |
+| experimental | SHALL be populated with {{ig experimental}} |
+| publisher | SHALL be populated with {{ig publisher}} |
+| description | SHALL be populated with {{Library Description}} |
+| knowledgeRepresentationLevel | SHALL be populated with {{narrative & (computable | executable)}} |
+
+##### ** Publishable **
+
+* Published `active` Library resources SHALL conform to CRMIPublishableLibrary
+
+| Element | Guidance |
+| ---- | ---- |
+| identifier | ???? |
+| date | SHALL be populated with {{ig publish date}} |
+| contact | SHALL be populated with {{ig contact}} |
+| useContext | SHALL be populated with {{ig useContext}} & ???? Decision Support Logic? Scheduling Logic? Indicator Logic? Data Element Logic? Concepts? Common? Settings? Configuration? |
+| jurisdication | SHALL be populated with {{ig jurisdiction}} |
+| purpose | SHALL be populated with {{Library Purpose}} |
+| copyright | SHALL be populated with {{ig copyright}} |
+| copyrightLabel | SHALL be populated with {{ig copyrightLabel}} |
+| approvalDate | SHALL be populated with {{ig approvalDate}} |
+| lastReviewDate | SHALL be populated with {{Library Review Date}} |
+| effectivePeriod | SHALL be populated with {{ig effectivePeriod}} |
+| topic | SHALL be populated with ???? |
+| author | SHALL be populated with {{ig author}} |
+| editor | SHOULD be populated with {{ig editor}} |
+| reviewer | SHOULD be populated with {{ig reviewer}} |
+| endorser | SHOULD be populated with {{ig endorser}} |
+| relatedArtifact | SHALL be populated with references to L2 content |
```

Comment:
With "citation" (may also have other inferred related artifacts, see CQLModule)

Reply:
We should be specific about how we reference L2 content from any relatedArtifact

Reply:
```suggestion
| relatedArtifact | SHALL be populated with `citation` references to L2 content |

TODO: Define how references to L2 content occur (not only for libraries but throughout)
```